lib/elixir/test/elixir/macro_test.exs

```diff
@@ -776,7 +776,7 @@ defmodule MacroTest do
       Macro.pipe(1, {:ok}, 0)
     end
 
-    assert_raise ArgumentError, ~r"cannot pipe 1 into 1 \+ 1", fn ->
+    assert_raise ArgumentError, ~r"cannot pipe 1 into 1 \+ 1, piping into a unary", fn ->
```

Comment:
Oh! I approved too fast. This assertion should not be "piping into a unary". it should say that the `+` operator can only take two arguments. To do that you can check the length of the args.

Reply:
Okay!

I just pushed a change.  Let me know what you think.